### ISSUE TITLE
Triple click selection doesn't work

### ISSUE THREAD

**Issue by Demetri0** [johnfactotum/foliate#1409]
**Describe the bug**
It is convenient that triple-click (LMB) selects a paragraph, but in Foliate it is doing nothing (_it seems like double click selects a word, and 3rd click deselect it_)

**To Reproduce**
1. Open any e-book (fb2 in my case, but it should not be matter)
2. Hover on some word within paragraph of text
3. Use triple click of LMB (3 fast click in a row, like a double click but + one more ;))
4. See that paragraph haven't been selected

**Expected behavior**
Full text of paragraph should be selected.

**Version:**
 - Foliate version: **feba39edd1203cafdb5ca112bbb8b4955a0e0be2** (`gtk4` branch of Sep 1 2024)
 - OS/Distribution and version: **Ubuntu 23.10 mantic**
 - Desktop environment: **GNOME 45.0**
 - Installation method: git [#run-without-building-or-installing](https://github.com/johnfactotum/foliate?tab=readme-ov-file#run-without-building-or-installing)

**Comment by johnfactotum** [comment]
Yes, it's quite unfortunate. The cause is, I think, that the selection popover is modal. Probably the selection popover should be made smaller, like only a single row. Then it can be more easily made non-modal. The popover would need to be non-modal, anyway, if one is to implement selection handles (#207).

**Comment by Jopp-gh** [comment]
Usually double click selects a word and triple click a sentence/paragraph. 

One way could be to add more delay before the popover pops up.

Or to open the popover only on right mouse click, but I guess this would not fit in the design choices of Foliate.
